docs/dockstore-logo-usage-guidelines.rst

```diff
@@ -0,0 +1,129 @@
+.. |Dockstore Logo Zip| image:: /assets/images/Dockstore_Logos_and_Icons.zip
+
+.. |Dockstore Dot| image:: /assets/images/Dockstore-blue.png
+  :width: 64
+
+.. |Vertical Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-vertical-blue.png
+  :width: 128
+
+.. |Vertical White| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-vertical-white.png
+  :width: 128
+
+.. |Vertical Black| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-vertical-black.png
+  :width: 128
+
+.. |Horizontal Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-horizontal-blue.png
+  :width: 256
+
+.. |Horizontal White| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-horizontal-white.png
+  :width: 256
+
+.. |Horizontal Black| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-horizontal-black.png
+  :width: 256
+
+.. |Vertical White Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-vertical-white-blue.png
+  :width: 128
+
+.. |Vertical Black Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-vertical-black-blue.png
+  :width: 128
+
+.. |Horizontal White Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-horizontal-white-blue.png
+  :width: 256
+
+.. |Horizontal Black Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-logo-horizontal-black-blue.png
+  :width: 256
+
+.. |Icon Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-blue.png
+  :width: 64
+
+.. |Icon Black| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-black.png
+  :width: 64
+
+.. |Icon White| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-white.png
+  :width: 64
+
+===============================
+Dockstore Logo Usage Guidelines
+===============================
+
+About
+=====
+It is important to establish a sense of branding throughout our publicly facing, digital media. Currently, we have several themes / layouts, navigation systems, font sets, color palettes. We would like to unify our designs under a single set of standards.
```

Comment:
I'd suggest:
> Currently, we have several themes, layouts, navigation systems, font sets, and color palettes.

Reply:
Changed paragraph.

Reply:
This whole section is presumably for external, non-Dockstore sites, but this is talking about "our" media, our themes, and how we aspire to unify them. While it's true that we want to unify them, I don't understand why we're talking about that here.

I would instead just say something like, very roughly/poorly, needs a lot of rework: Do you want to acknowledge or credit Dockstore on your site? Here are some guidelines for..., etc.

Or maybe I'm misunderstanding the intent of this ticket.